CHANGES.md

```diff
@@ -24,6 +24,40 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.15.0
+
+- new features
+  - ## Model Analysis
+    - move raw data into /data directory
+    - perform schema validations during compute and load
+    - rename model analysis to rai insights
+    - rename ModelAnalysisDashboard to ResponsibleAIDashboard
+    - directory based state management for model analysis tools
+    - add .json extension to train and test dataset
+  - ## Error Analysis
+    - port ErrorReport schema validation for error analysis to responsibleai
+    - add error analysis error importances (aka correlation of features with error) to model analyzer API
+    - add support for special classification outcome filter to error analysis backend to work with model analysis filters
+  - ## Counterfactual
+    - remove save as new data point option for read-only mode on What-if counterfactual
```

Comment:
I would add this change as well :- "Split counterfactual output into individual json files"

Reply:
yes I will have to keep updating it for now until all necessary PRs have been merged

Reply:
done!